element-connector/src/main/java/org/eclipse/californium/elements/config/Configuration.java

```diff
@@ -1550,16 +1593,20 @@ public static Configuration createFromStream(InputStream inStream, DefinitionsPr
 	 * If the provided file exists, the configuration reads the properties from
 	 * this file. Otherwise it creates the file.
 	 * 
-	 * Apply all {@link DefinitionsProvider} of registered modules.
+	 * Apply all {@link ModuleDefinitionsProvider} of registered modules.
 	 *
 	 * For Android, please use
 	 * {@link Configuration#createStandardWithoutFile()}, or
 	 * {@link Configuration#createStandardFromStream(InputStream)}.
 	 * 
 	 * @param file the configuration file
 	 * @return the standard configuration
+<<<<<<< cur
 	 * @throws NullPointerException if the file is {@code null}.
 	 * @see #addModule(String, DefinitionsProvider)
+=======
+	 * @see #addModule(ModuleDefinitionsProvider)
+>>>>>>> dbed5f2 Add Configuration with specific modules.
```

Comment:
same here

Reply:
yep. I rebased and fixed it.  Let me test it, before I push it.